models.py

```diff
@@ -22,3 +25,15 @@ def forward(self, x):
         x = x.view(-1, 784)
         x = self.layers(x)
         return F.log_softmax(x, dim=1)
+
+class HuggingFaceModel:
+
+    def __init__(self, model_name: str = "zephyr/zephyr-7b-beta") -> None:
+        if os.path.exists(f"/data/public_models/{model_name}"):
```

Comment:
Consider adding a comment that this is specific to the CAIS cluster

Reply:
Added comment and logger.info call